pkgs/development/libraries/gsignond/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, pkgconfig, meson, ninja, glib, glib-networking, sqlite, gobjectIntrospection, vala_0_34 }:
+
+stdenv.mkDerivation rec {
+  pname = "gsignond";
+  version = "1.0.7";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = pname;
+    rev = version;
+    sha256 = "03a9d92hldav18bhx63kgi88li8fnm56337bjsr7391awq1k9mv7";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja gobjectIntrospection vala_0_34 ];
+
+  buildInputs = [ glib glib-networking sqlite ];
+
+  mesonFlags = [
+    # Building documentation does not work in sandboxed environments
+    "-D documentation=false"
```

Comment:
should not have the space.

Reply:
Actually, `gtk-doc docbook_xsl docbook_xml_dtd_43 docbook_xml_dtd_45` should be enough to build the documentation.

Reply:
Ah I see, I'll add them.

Reply:
You will also need to add `glibcLocales` to `nativeBuildInputs` and `LC_ALL = "en_US.UTF-8";` to the attribute to prevent weird encoding error during docs build.

Reply:
hah I just saw that errors and was going to do exactly that.